frontend/src/pages/roles/components/ViewUserWindow.tsx

```diff
@@ -0,0 +1,72 @@
+import RoleDTO from '@shared/dtos/RoleDTO';
+import UserDTO from '@shared/dtos/UserDTO';
+import { useUserContext } from '../../../contexts/UserContext';
+import { UserRoleRow } from './UserRoleRow';
+
+export function ViewUserWindow({
+    users,
+    setUsers,
+    roles,
+}: {
+    users: Record<string, UserDTO>;
+    setUsers: (users: Record<string, UserDTO>) => void;
+    roles: Record<string, RoleDTO>;
+}) {
+    const content = Object.keys(users).map((username) => {
+        return {
+            tabTitle: users[username].name,
+            tabData: { ...users[username] },
+        };
+    });
+
+    const userContext = useUserContext();
+    async function removeRole(role: RoleDTO, userId: string) {
+        if (['Admin', 'Default'].indexOf(role.name) !== -1) {
```

Comment:
Thoughts on changing this to
``` 
    if (['Admin', 'Default'].includes(role.name)) {
        return;
    }
```

Mostly because it is a lot readable. Though, we may have to take a slight performance hit, though it is so small. Thus IMO readability > performance

Reply:
done